contracts/ENSRoot.sol

```diff
@@ -0,0 +1,99 @@
+pragma solidity ^0.4.18;
+
+import './ENS.sol';
+
+/**
+ * The ENS registry contract.
+ */
+contract ENSRoot is ENS {
+    struct Record {
+        address owner;
+        address resolver;
+        uint64 ttl;
+    }
+
+    mapping (bytes32 => Record) records;
+
+    // Permits modifications only by the owner of the specified node.
+    modifier only_owner(bytes32 node) {
+        require(records[node].owner == msg.sender);
+        _;
+    }
+
+    /**
+     * @dev Constructs a new ENS registrar.
+     */
+    function ENSRoot() public {
+        records[0].owner = msg.sender;
```

Comment:
There's a solidity linter error here complaining about left-justified decimal constants (oh no, a left padded zero!) - could you add an explicit cast to `bytes32`?

Reply:
If that works, sure.